File: src/main/java/com/example/solidaryaction/entities/Ong.java
package com.example.solidaryaction.entities;

import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Ong {


    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    public Long id;

    public String nomeFantasia;

    public String razaoSocial;

    public String cnpj;

    public String email;

    public String endereco;

    public int totalArrecadado;
}

File: front/app/types/ong.ts
export class Ong{
    constructor(
        public id: number | null,
        public nomeFantasia:string,
        public razaoSocial:string,
        public email:string,
        public cnpj:string,
        public endereco:string,
        public totalArrecadado:number, // no form é 0 
        public status:string,
    ){}
}

// prova - criar objeto
export interface OngFormProps{
    ongExistente?:Ong
}